.ci/docker/common/install_onnx.sh

```diff
@@ -24,7 +24,7 @@ pip_install \
   transformers==4.25.1
 
 # TODO: change this when onnx-script is on testPypi
-pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@2bb3e9f2d094912f81cb63cecb412efb14c65738"
+pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@ab54cb17feb256ca52fa4c616e27e06b3ce67139"
```

Comment:
Is this intended? My understanding was this was a diagnostic refactoring only

Reply:
We need the update in onnxscript to work with OPtionalType in torchscript (value: None).